<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qttree</class>
 <widget class="QMainWindow" name="qttree">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>573</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Tree Health</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>7</y>
      <width>991</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,2">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,0,0,0,0,0,0,0">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="font">
          <font>
           <family>Wide Latin</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>树木信息--------------------------------------</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget_3">
         <column>
          <property name="text">
           <string>树木编号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>树木种类</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>年龄</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>产地</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>保护等级</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="font">
          <font>
           <family>Wide Latin</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>图片信息--------------------------------------</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>图片位置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_3">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="0,1">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="text">
            <string>树木编号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_5"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="0,1">
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>诊断内容</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_6"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>开始诊断</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_18">
         <property name="font">
          <font>
           <family>Wide Latin</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>诊断结果--------------------------------------</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="textBrowser_6"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGraphicsView" name="graphicsView">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="foregroundBrush">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QPushButton" name="pushButton_6">
           <property name="text">
            <string>画笔</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_8">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_7">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="commandLinkButton">
           <property name="text">
            <string>清除所有(Ctrl+l)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1002</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_4"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_Ctrl_S">
   <property name="text">
    <string>保存(Ctrl+S)</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>测试</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>训练</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>诊断</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
